README.adoc

```diff
@@ -39,22 +39,64 @@ TIP: Use https://github.com/creationix/nvm/blob/master/README.md[nvm] to manage
 :numbered:
 === Clone the project
 
-First you need to clone the https://github.com/integr8ly/tutorial-web-app-walkthroughs[walkthroughs repository] next to the tutorial web app
+First you need to clone the https://github.com/integr8ly/tutorial-web-app-walkthroughs[walkthroughs repository] next to the tutorial web app:
 
 [source,shell]
 ----
-$ git clone https://github.com/integr8ly/tutorial-web-app-walkthroughs.git
+home:$ git clone https://github.com/integr8ly/tutorial-web-app-walkthroughs.git
```

Comment:
I think we should remove the prompt part of this `home:*` so that a user can just copy the commands without thinking.

Reply:
Yeah, that makes sense.